packages/fxa-settings/src/components/CardHeader/index.stories.tsx

```diff
@@ -0,0 +1,41 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import React from 'react';
```

Comment:
Technically, Card and Header are both nouns 🤭 It could go both ways, depending on what we want to group. I could see things like CardHeader, CardFooter, CardWhatever; FlowContainer, FlowHeader, etc.